promscale/installation/debian-ubuntu.md

```diff
@@ -0,0 +1,39 @@
+# Install Promscale on Debain or Ubuntu
+You can install Promscale on any Debian or Ubuntu system where you have an
+already installed and working Prometheus or OpenTelemetry environment.
+
+## Install TimescaleDB and Promscale extension
+To use with Promscale you need TimescaleDB database, as well as the Promscale
+extension. The Promscale extension contains support functions to improve
+performance of Promscale.
+
+<procedure>
+
+### Installing TimescaleDB and Promscale extension
+
+1.  Install TimescaleDB following the instructions in the
+    [TimescaleDB install page][tsdb-install-self-hosted]
+1.  Install Promscale extension
+    ```
+    apt install promscale-extension-postgresql-14
+    ```
+    
+</procedure>
+
+
+## Install the Promscale connector
+The Promscale connector understands PromQL queries natively and fetches data
```

Comment:
```suggestion
The Promscale connector consumes PromQL queries natively and fetches data
```

?? Not sure which verb to use here, but "understands" feels a little too much like anthropomorphism 😉